scitt/check_operation_status.py

```diff
@@ -0,0 +1,46 @@
+""" Module for checking when a statement has been anchored in the append-only ledger """
+
+import json
+import subprocess
+import argparse
+import time
+
+
+def check_operation_id(
+    operation_id: str
+)-> str:
+
+    return subprocess.check_output("curl -s -H @$HOME/.datatrails/bearer-token.txt https://app.datatrails.ai/archivist/v1/publicscitt/operations/"+operation_id, shell = True).decode()
+
+
+def main():
+    """Creates a signed statement"""
+
+    parser = argparse.ArgumentParser(description="Create a signed statement.")
+
+    # signing key file
+    parser.add_argument(
+        "--operation-id",
+        type=str,
+        help="the operation-id from a registered statement",
+    )
+
+    args = parser.parse_args()
+
+    # Check the operation status until the status=succeeded
+    while True:
```

Comment:
We need to ensure we don't poll forever, we need a hard time limit to just give up the polling.